NestedAccordion/js/scripts.js

```diff
@@ -0,0 +1,11 @@
+/* This is what I see:
```

Comment:
like we saw in class, would be great to see you break down the problem into baby steps and think about what JavaScript you might need to solve the problem.